state/src/replication/raft.rs

```diff
@@ -41,6 +41,8 @@ const DEFAULT_PROMOTION_TIMEOUT_MS: u64 = 5 * 60 * 1000;
 const DEFAULT_LEADER_ELECTION_TIMEOUT_MS: u64 = 30_000; // 30 seconds
 /// The default max chunk size for snapshots
 const DEFAULT_SNAPSHOT_MAX_CHUNK_SIZE: u64 = 10 * 1024 * 1024; // 10MiB
+/// The default timeout to use when sending `InstallSnapshot` RPCs
+const DEFAULT_INSTALL_SNAPSHOT_TIMEOUT_MS: u64 = 1000; // 1 second
```

Comment:
I'd bump this, maybe 3 or 5 seconds? Give a little extra time seeing as these snapshot chunks are somewhat big